pom.xml

```diff
@@ -177,16 +181,37 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
+                <version>3.0.0</version>
             </plugin>
+			<plugin>
+				<groupId>io.github.git-commit-id</groupId>
+				<artifactId>git-commit-id-maven-plugin</artifactId>

```

Comment:
This plugin is [LGPLv3](https://fossa.com/blog/open-source-software-licenses-101-lgpl-license/). 
According to the license, 

> Derivative works of LGPL v3 code must be licensed under the LGPL v3 — or GPL v3 or later. You would not be able to license a derivative work of LGPL v3 under GPL v2 or LGPL 2.1

Now, my assumption is that since this plugin is used to build our SDK, but it's not distributed or linked to it in any way, it does classify as derivative work and so our Apache 2 license won't be compromised. Could you confirm?

Reply:
This is correct.

Reply:
If I understand correctly, this will allow having build numbers tied to the git sha for all versions in between releases, is that correct?

Reply:
Correct. The given sha1 of a commit will be added as a JAR manifest attribute. That way we can tie commit time + commit sha + release version as part of the artifact's metadata.